modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,196 @@
+
+* This update adds a new command named `tkn pac info` to the {pac} CLI. As an administrator, you can use the `tkn pac info` command to obtain the following details about the {pac} installation:
+** The location where {pac} is installed.
+** The version number of {pac}.
+** An overview of the `Repository` CR created on the cluster and the URL associated with the repository.
+** Details of any installed GitHub applications.
++
+With this command, you can also specify a custom GitHub API URL by using the `--github-api-url` argument.
+
+* With this update, error detection is enabled for all `PipelineRuns` by default. {pac} detects if a `PipelineRun` execution has failed and shows a snippet of the last few lines of the error. For a GitHub application, {pac} detects error messages in the container logs and exposes them as annotations on a pull request. 
+
+* This update allows you to fetch tasks from a private {tekton-hub} instance attached to a private Git repository. To enable this update, {pac} uses the internal raw URL of the private {tekton-hub} instance instead of using the GitHub raw URL.
+
+* Before this update, {pac} provided logs that would not include the namespace detail. With this update, {pac} adds the namespace information to the pipeline logs so that you can filter them based on a namespace and debug easily.
+
+* With this update, you can define the provenance source from where the `PipelineRun` definition will be fetched. By default, {pac} fetches the `PipelineRun` definition from the branch where the event has been triggered. Now, you can configure the value of the `pipelinerun_provenance` setting to `default_branch` so that the `PipelineRun` definition will be fetched from the default branch of the repository as configured on GitHub.
+
+* With this update, you can extend the scope of the GitHub token at the following two levels:
+** Repository-level: Use this level to extend the scope to the repositories that exist in the same namespace in which the original repository exists.
+** Global-level: Use this level to extend the scope to the repositories that exist in a different namespace.
+
+* With this update, {pac} triggers a CI pipeline for a pull request created by a user who is not an owner, collaborator, or public member or is not listed in the `owner` file but has permissions to push changes to the repository.
+
+* With this update, the custom console setting allows you to use custom parameters from a `Repository` CR. 
+
+* With this update, {pac} changes all `PipelineRun` labels to `PipelineRun` annotations. You can use a `PipelineRun` annotation to mark a Tekton resource, instead of using a `PipelineRun` label.
+
+* With this update, {pac} provides an option to install a webhook forwarder named `gosmee`. This forwarder enables connectivity between the {pac} controller and GitHub without requiring an internet connection. You need to explicitly specify the `--force-gosmee` flag when bootstrapping on the {product-title} cluster.
```

Comment:
we can remove this, feature is not working and there is something else we need to talk about it to make it customer facing